docs/api.rst

```diff
@@ -93,9 +93,12 @@ The KSQL resource runs a sequence of KSQL statements. All statements, except tho
    :>json string  sourceDescription.name: The name of the stream or table.
    :>json array   sourceDescription.readQueries: The queries reading from the stream or table.
    :>json array   sourceDescription.writeQueries: The queries writing into the stream or table
-   :>json array   sourceDescription.schema: The schema of the stream or table as a list of column names and types.
-   :>json string  sourceDescription.schema[i].name: The name of the column.
-   :>json string  sourceDescription.schema[i].type: The data type of the column.
+   :>json array   sourceDescription.fields: The schema of the stream or table as a list of column names and types.
+   :>json string  sourceDescription.fields[i].name: The name of the column.
```

Comment:
can we call out a `Field` type here? instead of `fields[i]`? I get 
```
Field = {
  name: string,
  schema: {
    type: Type, 
    memberSchema: MAP | ARRAY,
    fields: string
  }
`
is the actual type inside of fields.